internal/cmd/schema-registry/schema_utilities.go

```diff
@@ -30,6 +30,9 @@ type RegisterSchemaConfigs struct {
 	SchemaType  string
 	SchemaPath  *string
 	Refs        []srsdk.SchemaReference
+	Metadata    srsdk.Metadata
+	RuleSet     srsdk.RuleSet
```

Comment:
Nit: Ruleset is one word
```suggestion
	Ruleset     srsdk.RuleSet
```